src/commands/tip.js

```diff
@@ -30,29 +31,21 @@ function tip (message, dogecoinNode, amount) {
   var fromAccount = message.author.tag.replace('#', '')
   var toAccount = to.username + to.discriminator
 
-  dogecoinNode.getBalance(fromAccount, function (err, balance) {
-    if (err) {
-      console.log(err)
-      message.channel.send(OOPS_TEXT)
-      return
-    }
+  try {
+    const balance  = await getBalance(fromAccount)
```

Comment:
Faut que j'aille lire à ce sujet, je ne connais pas le fonctionnement de async et await.
C'est une autre façon de faire par rapport à des Promises ?

Reply:
Yes it will wait for the promise to be fulfill.